media-video/vlc/vlc-3.0.20-r1.ebuild

```diff
@@ -0,0 +1,517 @@
+# Copyright 2000-2023 Gentoo Authors
```

Comment:
update *9999* too

Reply:
Updated `vlc-3.0.9999` `vlc-9999` and `libsndfile-9999`